db/migrate/20220811093302_update_audits_generated_by_api_user.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# rubocop:disable Rails/ReversibleMigration
+class UpdateAuditsGeneratedByApiUser < ActiveRecord::Migration[6.1]
+  # rubocop:disable Metrics/MethodLength
+  def change
+    execute(
+      "UPDATE audits
+      SET api_user_id = 1
```

Comment:
Luckily there's only one API user record on preview, with id 1, and on production only 2 - id 1 is 'api_user', and id 2 is 'RIPA.